modules/dynamic-operator/plugins/docker-compose/plugin/compose/objects.go

```diff
@@ -345,6 +346,18 @@ func (c *ComposeModel) setApplicationObjects(req *commons.PluginRequest) error {
 				Protocol:      proto,
 			}
 			container.Ports = append(container.Ports, port)
+
+			container.ReadinessProbe = &corev1.Probe{
+				Handler: corev1.Handler{
+					HTTPGet: &corev1.HTTPGetAction{
+						Path: "/healthz",
+						Port: target,
```

Comment:
Port should be pointing to either container port name (see line 344) or port.ContainerPort (line 345). `target` can be used, but in this case, please make sure in tests that target.String() is pointing to port.Name (which is not) and target.IntVal equals to port.ContainerPort.

Reply:
Actually, you can transform `target.String() + "-port"` to `target.String()`

Reply:
Fixed

Reply:
I've tried to remove `-port` part, but ci failed instantly and I guess `-port` part actually doing something here, you can check failed test: https://github.com/kuberlogic/kuberlogic/runs/8243755117?check_suite_focus=true

Reply:
Fair enough